src/DynamoCoreWpf/Views/GuidedTour/PopupWindow.xaml

```diff
@@ -0,0 +1,152 @@
+﻿<Popup x:Class="Dynamo.Wpf.Views.GuidedTour.PopupWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
```

Comment:
For some of the inline styles in this file, shall we move them to DynamoModern.xml?

Reply:
@QilongTang I moved several styles to the DynamoModern.xaml